css/style.css

```diff
@@ -123,6 +124,129 @@ header{
     .nav-menu.active{
       left: 0;
     }
+}
+  
+
+  /* Home Section Code */
+
+.content{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0px;
+    gap: 80px;
+}
+
+.text{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.text p{
+    font-size: 20px;
+    color: #fff;
+}
+
+.text p span{
+    color: var(--secondary-color);
+}
+
+#welcome{
+    font-size: 40px;
+}
+
+#quote{
+    margin-left: 10px;
+    margin-bottom: 10px;
+}
+
+.text a{
+    position: relative;
+    display: inline-block;
+    font-size: 18px;
+    color: var(--light-color);
+    text-decoration: none;
+    padding: 18px 30px;
+    letter-spacing: 2px;
+    font-weight: 500;
+    border-radius: 50px;
+    background-color: var(--secondary-color);
+    transition: transform 0.3s ease;
+    box-shadow: 0px 3px 3px rgba(255, 255, 255, 0.2);
+}
+
+.text a:hover {
+    transform: scale(1.05);
+}
+
+.img{
+  z-index: 1;
```

Comment:
is this even working ?
zindex is useless without position property

Reply:
Yes, its working. 
That's, also true that z-index is useless without position property but it's working fine.